fighters/common/src/general_statuses/mod.rs

```diff
@@ -54,8 +54,7 @@ pub unsafe fn sub_wait_common_Main(fighter: &mut L2CFighterCommon) -> L2CValue {
 
 #[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_status_DamageAir_Main)]
 pub unsafe fn damage_air_main(fighter: &mut L2CFighterCommon) -> L2CValue {
-    // ControlModule::clear_command_one(fighter.module_accessor, *FIGHTER_PAD_COMMAND_CATEGORY1, *FIGHTER_PAD_CMD_CAT1_AIR_ESCAPE);
-    BufferModule::clear_persist_one(fighter.battle_object, *FIGHTER_PAD_COMMAND_CATEGORY1, *FIGHTER_PAD_CMD_CAT1_AIR_ESCAPE);
+    ControlModule::clear_command_one(fighter.module_accessor, *FIGHTER_PAD_COMMAND_CATEGORY1, *FIGHTER_PAD_CMD_CAT1_AIR_ESCAPE);
```

Comment:
if you'd like we can change the implementation of BufferModule to call this function too

Reply:
would be nice in the long run, but i commented this hook install out anyway, wasnt needed